conversion-measurement/functions/views/advertiser/add-to-cart.pug

```diff
@@ -2,6 +2,6 @@ extends _layout.pug
 
 block content
     h3 Product added to cart 
-    img.pixel(height='12' width='12' attributionsrc=adtechRequestUrl)
+    img.pixel(height='12' width='12' src=adtechRequestUrl)
```

Comment:
To clarify: We're deliberately omitting the `attributionsrc` attribute here entirely? We may want to use valueless `attributionsrc` so that the image request contains the `Attribution-Reporting-Eligible` header, but it's certainly not required.

Same question about the corresponding changes in the other `.pug` files.